inference-engine/tests/functional/inference_engine/transformations/compare_functions_test.cpp

```diff
@@ -522,3 +522,42 @@ TEST(TransformationTests, DummyOpNegativeNotSupportedType) {
     EXPECT_FALSE(res.valid);
     EXPECT_THAT(res.message, HasSubstr(" [drop `void` comparison which is '"));
 }
+
+TEST(TransformationTests, DifferentPrecisionVersusAttributes) {
+    const auto createReadValueFunc = [](ngraph::element::Type t) {
+        using namespace ngraph::opset5;
```

Comment:
nitpick: opset5 -> opset6

Reply:
Do we update opsets in test? Is this general rule?

Reply:
we try to use the latest opset version everywhere.

Reply:
OK. got it.